### PR TITLE
Fix tab component find for ActionPreview

### PR DESCRIPTION
**Pull request by jhen0409**
Fixes the error `Uncaught TypeError: Cannot read property 'component' of undefined`.

### CODE REVIEW

**Comment by zalmoxisus** [comment]
It shouldn't happen, because [we're preventing the first rendering till the monitor reducer gets invoked](https://github.com/zalmoxisus/remotedev-app/blob/master/src/app/containers/DevTools.js#L57-L60). Could you please provide steps to reproduce?

Of course, merging this PR will help to avoid that workaround, but the same problem will be for Chart Monitor and for other monitors as well. Monitor's authors expect the reducer to be called with default values before rendering the component, but, in case of the extension, we need first to render the component before being able to call the reducer (to call `DevtoolsInspector.update`).

**Comment by jhen0409** [comment]
A reason is I used `redux-persist`, and it persist the older monitorState (No `tabName`), so this error will thrown when `persist/REHYDRATE` dispatched.

Also I changed to `defaultProps`.

File: src/ActionPreview.jsx [hunk 1]
```diff
@@ -16,6 +16,10 @@ const DEFAULT_TABS = [{
 }]
 
 class ActionPreview extends Component {
+  static defaultProps = {
+    tabName: 'Action'
```
Comment (zalmoxisus) [inline src/ActionPreview.jsx]: Shouldn't it be `Diff` like in [default state](https://github.com/alexkuz/redux-devtools-inspector/blob/master/src/redux.js#L8)?

**Review by zalmoxisus — CHANGES_REQUESTED** [review]

File: src/ActionPreview.jsx (hunk 1 above)
Comment (jhen0409) [reply src/ActionPreview.jsx]: Ahh you're right, It's fixed.

**Review by zalmoxisus — APPROVED** [review]

**Pull request merged**